### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by briankgarland** [SinestroWhite/Normalize-JSON-API#5]
Can this be used with a service that pulls in data from an API so that this:

{
    "Person": [
        {
            "Person_ID": "0000",
            "First_Name": "Tamara",
            "Last_Name": "Truck",
            "Employee_Status_Description": "Terminated",
            "Location_Country_2Char_Code": "",
            "Business_Phone_number": "",
            "BusinessMobile_Phone_number": "",
            "Business_Email": "",
            "Job_Description": "Assistant Distribution Center Mgr"
        },
        {
            "Person_ID": "TEST1",
            "First_Name": "Employee",
            "Last_Name": "One",
            "Employee_Status_Description": "Active",
            "Location_Country_2Char_Code": "",
            "Business_Phone_number": "1098765432",
            "BusinessMobile_Phone_number": "",
            "Business_Email": "eric.wojnar@awesome.com",
            "Job_Description": ""
        }
   ]
}

Just becomes this:

        {
            "Person_ID": "0000",
            "First_Name": "Tamara",
            "Last_Name": "Truck",
            "Employee_Status_Description": "Terminated",
            "Location_Country_2Char_Code": "",
            "Business_Phone_number": "",
            "BusinessMobile_Phone_number": "",
            "Business_Email": "",
            "Job_Description": "Assistant Distribution Center Mgr"
        },
        {
            "Person_ID": "TEST1",
            "First_Name": "Employee",
            "Last_Name": "One",
            "Employee_Status_Description": "Active",
            "Location_Country_2Char_Code": "",
            "Business_Phone_number": "1098765432",
            "BusinessMobile_Phone_number": "",
            "Business_Email": "eric.wojnar@awesome.com",
            "Job_Description": ""
        }

If so, what does the code look like? In the service or the app component?

**Comment by SinestroWhite** [comment]
Is the first object pasted above the complete response from the API? If no, can you provide a more complete example?

**Comment by briankgarland** [comment]
Yes, it is. But we haven’t decided to just write our own custom code. Thanks for the reply.
